src/sources/splunk_hec/acknowledgements.rs

```diff
@@ -27,28 +27,32 @@ pub struct HecAcknowledgementsConfig {
     /// Enables end-to-end acknowledgements.
     pub enabled: Option<bool>,
 
-    /// The maximum number of ack statuses pending query across all channels.
+    /// The maximum number of acknowledgement statuses pending query across all channels.
     ///
     /// Equivalent to the `max_number_of_acked_requests_pending_query` Splunk HEC setting.
     ///
     /// Minimum of `1`.
+    #[configurable(metadata(docs::human_name = "Max Number of Pending Acknowledgements"))]
     pub max_pending_acks: NonZeroU64,
 
     /// The maximum number of Splunk HEC channels clients can use with this source.
     ///
     /// Minimum of `1`.
+    #[configurable(metadata(docs::human_name = "Max Number of  Acknowledgement Channels"))]
```

Comment:
Double space between `of` and `Acknowledgement`.

Reply:
Fixed